### DISCUSSION TITLE
Micropython on RISC-V machines

### DISCUSSION THREAD

**Discussion by FathiMahdi** [micropython/micropython#15267] [General]
Hello, I am wondering if Micropython can support RISC-V boards like the CH32V203 microcontroller.

**Comment by GitHubsSilverBullet** [comment]
Both, RAM and FLASH are quite likely to small for any useful work.
But I'm with you, a RISC-V µC with hardware comparable to the RP2040 would be awesome.

**Comment by mattytrentini** [comment]
Please see the work-in-progress PR to [add support for the CH32V203](https://github.com/micropython/micropython/pull/8637)
